odoo/addons/base/models/ir_session.py

```diff
@@ -0,0 +1,22 @@
+from datetime import timedelta, datetime
```

Comment:
February 2022 update: after internal discussions, the session have been moved back on the filestore. This commit have not been amended because it is too deep in the historic.

Reply:
Hello @Julien00859 , the PR has now been merged for a little eternity but somehow I thought that the session is now stored in the database. Especially for setups on a cloud that would have been very handy. What is the reason why you went back to the filestore. And is the implementation of the database still available somewhere for reference? Did you store the session per database or in a separate session database (like https://github.com/odoo/odoo-extra/tree/master/session_db) for all databases? If it was per database how did the request find the right database again without a sessionstore file? Would be great to get some insight. Thanks for the great PR :)

Reply:
I've created a new issue #104503 to answer your question, discussion continues over there :)